app/src/main/java/com/example/alpaki/di/ApplicationModule.kt

```diff
@@ -9,6 +10,7 @@ import dagger.hilt.android.components.ApplicationComponent
 @Module(
     includes = [
         DataModule::class,
+        DataBaseModule::class,
```

Comment:
Popraw proszę tę wielką literę w środku (database jako jedno słowo)

Reply:
Też: widzę, że ten sam moduł masz dodany w DataModule. Jeżeli ApplicationModule ma już DataModule, to nie musisz dodawać tu tego modułu (będzie dostarczony przez DataModule z automatu)

Reply:
poprawione